eng/pipelines/templates/jobs/tests-nightly-python.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         continueOnError: true
 
       - task: PublishTestResults@2
-        condition: succeededOrFailed()
+        condition: always()
```

Comment:
It is probably worth updating our devops guidelines wiki with information about using "always".